src/core/thread.d

```diff
@@ -3660,8 +3668,17 @@ shared static this()
     }
     else version (Posix)
     {
-        PAGESIZE = cast(size_t)sysconf(_SC_PAGESIZE);
-        PTHREAD_STACK_MIN = cast(size_t)sysconf(_SC_THREAD_STACK_MIN);
+        version (OpenBSD)
+        {
+            static import core.sys.openbsd.sys.types;
+            PAGESIZE = cast(size_t)core.sys.openbsd.sys.types.PAGE_SIZE;
+            PTHREAD_STACK_MIN = cast(size_t)(1U << core.sys.openbsd.sys.types._MAX_PAGE_SHIFT);
+        }
```

Comment:
I suggest putting this `OpenBSD` block outside the `Posix` block. Will reduce indentation.

Reply:
Fixed.